src/fhirtypes/common.ts

```diff
@@ -83,3 +86,23 @@ export function getArrayIndex(pathPart: PathPart): number {
   }
   return arrayIndex >= 0 ? arrayIndex : null;
 }
+
+/**
+ * Replaces references to instances by the correct path to that instance
+ * @param {FixedValueRule} rule - The rule to replace references on
+ * @param {FSHTank} tank - The tank holding the instances
+ * @param {ResolveFn} resolve - A function that can resolve to Structure Definitions
+ */
+export function replaceReferences(rule: FixedValueRule, tank: FSHTank, resolve: ResolveFn): void {
```

Comment:
Do you think maybe we should clone the rule, replace references in the clone, and then return the cloned rule?  It feels a little weird for the exporter to actually mutate things in the tank at this point.  I think I'd be more comfortable leaving things in the tank as-is if possible.